src/components/page-environment/environment-overview.js

```diff
@@ -226,7 +226,7 @@ export class EnvironmentOverview extends Component {
                     ) : (
                       <Typography>No active deployment</Typography>
                     )}
-                    {deployment.gitTags && (
+                    {deployment && deployment.gitTags && (
```

Comment:
same check, just shorter
```suggestion
                    {deployment?.gitTags && (
```